autoPyTorch/pipeline/components/setup/network_backbone/ConvNetImageBackbone.py

```diff
@@ -0,0 +1,127 @@
+from typing import Any, Dict, List, Optional, Tuple
+
+import ConfigSpace as CS
+from ConfigSpace.configuration_space import ConfigurationSpace
+from ConfigSpace.hyperparameters import (
+    CategoricalHyperparameter,
+    UniformIntegerHyperparameter
+)
+
+from torch import nn
+
+from autoPyTorch.pipeline.components.setup.network_backbone.base_network_backbone import NetworkBackboneComponent
+
+_activations: Dict[str, nn.Module] = {
```

Comment:
Why not import it from [here](https://github.com/automl/Auto-PyTorch/blob/ddc0f3d890c5a377132108ea03cb23bc093ce17e/autoPyTorch/pipeline/components/setup/network_backbone/ShapedMLPBackbone.py#L17)

Reply:
Right, I will change it